### ISSUE TITLE
A11y_code.visualstudio.com_DocsSetup_MacOS_ScreenReader: Incorrect reading order announced to the screen reader users in the main content.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-docs#2778]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-manai%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: [https://code.visualstudio.com](https://code.visualstudio.com)
Windows Version: Windows10
Browser Version: 
Microsoft Edge 42.17134.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated: MAS 1.3.2

**Repro Steps:**
Navigate to the url: [https://code.visualstudio.com/docs/setup/mac](https://code.visualstudio.com/docs/setup/mac)
follow below steps:
1.	Launch the application Url: [https://code.visualstudio.com](https://code.visualstudio.com)
2.	Navigate to "Docs" link.
3.	From left navigation, select "Setup".
4.	Navigate to "MacOS" link and activate it.
5.	Navigate to the main content.
6.	Start the screen reader.
7.	Navigate to the “Edit” link and to “Visual Studio Code on macOS” heading element.
8.	Observe the visually sequence of these content and listen if the same is announced or not.

**Actual:**
When screen reader users navigate to the main content “Edit” link  is announced prior to the “Visual Studio Code on macOS” heading element which is incorrect as visually it appears the heading element should be before the “Edit” link.

Similar issue was observed with "Edge + Narrator" as well as with "Chrome + NVDA".

**Expected:**
When screen reader users navigate to the main content first the “Visual Studio Code on macOS” heading element then the “Edit” link followed by the rest of the content should be announced to the screen reader users.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
If the visual sequence of any content present on the screen is not the same that is announced to the screen reader users then they will not get the same information as it is conveyed to the visual users.

**MAS Reference:**
MAS 1.3.2 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b} )

**Attachment for Reference:**
[Code.visualstudio.com_Docs Setup_MacOS_ScreenReader Incorrect reading order.pptx](https://github.com/microsoft/vscode-docs/files/3313795/Code.visualstudio.com_Docs.Setup_MacOS_ScreenReader.Incorrect.reading.order.pptx)

**Comment by gregvanl** [comment]
The Edit button is part of the topic border and comes before the actual content even though visually lined up with the H1 text for better page real estate use.

**Comment by MayuriNaik** [comment]
Similar issue is been observed in following scenarios:
1) Themes - [https://code.visualstudio.com/docs/getstarted/themes](https://code.visualstudio.com/docs/getstarted/themes)
2) Continuous Integration - [https://code.visualstudio.com/api/working-with-extensions/continuous-integration](https://code.visualstudio.com/api/working-with-extensions/continuous-integration)
3) Display Language - [https://code.visualstudio.com/docs/getstarted/locales](https://code.visualstudio.com/docs/getstarted/locales)
4) Angular Tutorial - [https://code.visualstudio.com/docs/nodejs/angular-tutorial](https://code.visualstudio.com/docs/nodejs/angular-tutorial)
5) Java - [https://code.visualstudio.com/docs/languages/java](https://code.visualstudio.com/docs/languages/java)
